src/functions/searchResults/framework/database/query-builder.ts

```diff
@@ -22,9 +22,16 @@ export const getConciseSearchResultsFromSearchQuery = (queryParameters : QueryPa
     queries.push('staff_number = ?');
     parameterArray.push(queryParameters.staffNumber);
   }
+
   if (queryParameters.applicationReference) {
-    queries.push('application_reference = ?');
-    parameterArray.push(queryParameters.applicationReference.toString());
+    if (queryParameters.applicationReference.toString().length === 8) {
+      queries.push('application_reference >= ? AND application_reference <= ?');
```

Comment:
Can you add a comment around why we have to do it this way and not using a LIKE or other operator